UnitTest.Commands/source/empty.cpp

```diff
@@ -0,0 +1 @@
+2020-04-06 14:15:16 1 ERROR - AppInsLog - Output file not specified for custom rules pack command
```

Comment:
Same as others, why is this being checked in?

Reply:
Removed unneeded versions of these files which already exists in the desired locations under source\unzipped\simple